pkg/wharfyml/step.go

```diff
@@ -0,0 +1,26 @@
+package wharfyml
+
+import (
+	"fmt"
+)
+
+type Step struct {
+	Name      string
+	Type      StepType
+	Variables map[string]interface{}
+}
+
+func parseStep(name string, content map[string]interface{}) (Step, error) {
+	if len(content) != 1 {
+		return Step{}, fmt.Errorf("expected single step-type, got %d", len(content))
+	}
+
+	var stepType StepType
+	var variables map[string]interface{}
+	for k, v := range content {
+		stepType = ParseStepType(k)
+		variables = v.(map[string]interface{})
+	}
+
+	return Step{Name: name, Type: stepType, Variables: variables}, nil
+}
```

Comment:
Here is added code. The title is "Removed unused code" whats up?

Reply:
It was moved from `pkg/core/wharfyml/step.go` (https://github.com/iver-wharf/wharf-cmd/pull/44/files/039ad277d80730b219cc20ce993ca046c8f7d709#diff-df80e6fb3d9eeac827bd9bd4cc5de251d3e5f6a28834c56fdae068899cf4f5c0) but I also removed some functions related to `containercreator`. Slightly annoyed that it didn't detect that it's both a file rename and some changes.